### PR TITLE
Introduce maxSdk sample & fix tests

### PR DESCRIPTION
**Pull request by keyboardsurfer**

### CODE REVIEW

File: DynamicFeatures/app/src/main/res/values-pl/strings.xml [hunk 1]
```diff
@@ -26,6 +26,7 @@
     <string name="instructions">Naciśnij przycisk, aby wywołać akcję</string>
     <string name="load_all_features_deferred">Załaduj wszystkie funkcje później</string>
     <string name="load_assets">Pokaż assety</string>
+    <string name="start_maxsdk">TODO Translate</string>
```
Comment (wojtek-kalicinski) [inline DynamicFeatures/app/src/main/res/values-pl/strings.xml:29]:
```suggestion
    <string name="start_maxsdk">Uruchom funkcję z maxSdk</string>
```

File: DynamicFeatures/features/assets/build.gradle [hunk 2]
```diff
@@ -21,12 +21,16 @@ android {
     compileSdkVersion versions.compileSdk
 
     defaultConfig {
+
```
Comment (wojtek-kalicinski) [inline DynamicFeatures/features/assets/build.gradle:24]: why the empty lines after `{`, is it autoformatted like this?

File: DynamicFeatures/features/initialInstall/build.gradle [hunk 3]
```diff
@@ -22,9 +22,11 @@ android {
     compileSdkVersion versions.compileSdk
 
     defaultConfig {
+
```
Comment (wojtek-kalicinski) [inline DynamicFeatures/features/initialInstall/build.gradle:25]: and here

Comment (wojtek-kalicinski) [reply DynamicFeatures/features/initialInstall/build.gradle:25]: and everywhere

File: DynamicFeatures/features/maxSdk/src/main/res/values-pl/strings.xml [hunk 4]
```diff
@@ -15,5 +15,5 @@
   -->
 
 <resources>
-    <string name="content_text">To jest dynamiczna funkcja napisana w \nKotlinie\n, która może być załadowana na żądanie.</string>
+    <string name="content_text">TODO Translate</string>
```
Comment (wojtek-kalicinski) [inline DynamicFeatures/features/maxSdk/src/main/res/values-pl/strings.xml:18]:
```suggestion
    <string name="content_text">To jest dynamiczna funkcja, która będzie instalowana warunkowo tylko do ustawionej maksymalnej wersji SDK.</string>
```

File: DynamicFeatures/gradle.properties [hunk 5]
```diff
@@ -27,4 +27,4 @@ org.gradle.jvmargs=-Xmx1536m
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 org.gradle.parallel=true
-android.useAndroidX=true
+android.useAndroidX=true
```
Comment (wojtek-kalicinski) [inline DynamicFeatures/gradle.properties:30]: keep the new line

Comment (keyboardsurfer) [reply DynamicFeatures/gradle.properties:30]: Done

File: DynamicFeatures/features/initialInstall/build.gradle (hunk 3 above)
Comment (keyboardsurfer) [reply DynamicFeatures/features/initialInstall/build.gradle:25]: Reverted

File: DynamicFeatures/features/assets/build.gradle (hunk 2 above)
Comment (keyboardsurfer) [reply DynamicFeatures/features/assets/build.gradle:24]: Reverted

File: DynamicFeatures/app/src/main/res/values-pl/strings.xml (hunk 1 above)
Comment (wojtek-kalicinski) [reply DynamicFeatures/app/src/main/res/values-pl/strings.xml:29]: missed one @keyboardsurfer

Comment (keyboardsurfer) [reply DynamicFeatures/app/src/main/res/values-pl/strings.xml]: Added

**Pull request merged**